PCAxis.Common/Serializers/CsvFileSerializer.vb

```diff
@@ -262,22 +310,45 @@ Namespace PCAxis.Paxiom
             Dim sb As New System.Text.StringBuilder
 
             If Me.DoubleColumn Then
-                If _model.Meta.Stub(stubIndex).DoubleColumn Then
-                    If _model.Meta.Stub(stubIndex).Values(valueIndex).HasCode Then
+                If Model.Meta.Stub(stubIndex).DoubleColumn Then
+                    If Model.Meta.Stub(stubIndex).Values(valueIndex).HasCode Then
                         If Me.WrapTextWithQuote Then sb.Append(ControlChars.Quote)
-                        sb.Append(_model.Meta.Stub(stubIndex).Values(valueIndex).Code)
+                        sb.Append(Model.Meta.Stub(stubIndex).Values(valueIndex).Code)
                         If Me.WrapTextWithQuote Then sb.Append(ControlChars.Quote)
                         sb.Append(Me.Delimiter)
                     End If
                 End If
             End If
             If Me.WrapTextWithQuote Then sb.Append(ControlChars.Quote)
-            sb.Append(_model.Meta.Stub(stubIndex).Values(valueIndex).Text)
+            If Not UseShortDescription Then
+                sb.Append(Model.Meta.Stub(stubIndex).Values(valueIndex).Text)
+            Else
+                sb.Append(Model.Meta.Stub(stubIndex).Values(valueIndex).Code)
+            End If
             If Me.WrapTextWithQuote Then sb.Append(ControlChars.Quote)
 
             Return sb.ToString
         End Function
 
+        Protected Function GetPivotedModelAllStub(ByVal oldModel As PXModel) As Paxiom.PXModel
```

Comment:
Feels like this is misplaced, maybe this function should be extracted to its own utility class.

Reply:
Maybe in Misc-folder?

Reply:
Sounds like the right place.